components/Login/LoginButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "./LoginButton.module.css";
+
+function LoginButton() {
+  function handleClick(e) {
+    e.preventDefault();
+    console.log("You will be redirected to spotify.");
+  }
+
+  return (
+    <button className={`${styles.login}`}>
+      <img src="/spotify-icon.svg" className={`${styles.spotify}`} />
+      <a href="#" onClick={handleClick}>
+        Login with Spotify
+      </a>
+    </button>
```

Comment:
Is it better to place the `<a>` around the button? So the user can click anywhere at the button and not just at the text...

```HTML
<a>
   <button>
     <img>
   </button>
</a>
```

Reply:
Yes, i will change that!